ansible/setup.yml

```diff
@@ -84,6 +84,13 @@
       tags:
         - dev-tools
 
+    - name: Increase ulimit
+      pam_limits:
+        domain: epoch
+        limit_type: soft
```

Comment:
Can you please confirm that this step fails if hard limit is lower so this soft limit is ineffective?